runtime/moonbase/src/weights/pallet_moonbeam_lazy_migrations.rs

```diff
@@ -136,27 +140,31 @@ impl<T: frame_system::Config> pallet_moonbeam_lazy_migrations::WeightInfo for We
 	/// Proof: `Assets::Asset` (`max_values`: None, `max_size`: Some(174), added: 2649, mode: `MaxEncodedLen`)
 	/// Storage: `EVM::AccountStorages` (r:1002 w:1001)
 	/// Proof: `EVM::AccountStorages` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `Ethereum::Pending` (r:1 w:1)
-	/// Proof: `Ethereum::Pending` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `Ethereum::CounterForPending` (r:1 w:1)
+	/// Proof: `Ethereum::CounterForPending` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `Ethereum::Pending` (r:1001 w:1000)
+	/// Proof: `Ethereum::Pending` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// The range of component `n` is `[1, 1000]`.
 	fn migrate_foreign_asset_balances(n: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `24607 + n * (215 ±0)`
-		//  Estimated: `33179 + n * (2691 ±0)`
-		// Minimum execution time: 431_433_000 picoseconds.
-		Weight::from_parts(438_379_000, 33179)
-			// Standard Error: 8_575_117
-			.saturating_add(Weight::from_parts(870_965_895, 0).saturating_mul(n.into()))
-			.saturating_add(T::DbWeight::get().reads(19_u64))
-			.saturating_add(T::DbWeight::get().reads((3_u64).saturating_mul(n.into())))
-			.saturating_add(T::DbWeight::get().writes(8_u64))
-			.saturating_add(T::DbWeight::get().writes((3_u64).saturating_mul(n.into())))
+		//  Measured:  `24636 + n * (215 ±0)`
+		//  Estimated: `33208 + n * (2691 ±0)`
+		// Minimum execution time: 458_134_000 picoseconds.
+		Weight::from_parts(463_941_000, 33208)
+			// Standard Error: 62_378
+			.saturating_add(Weight::from_parts(172_099_937, 0).saturating_mul(n.into()))
```

Comment:
The big decrease here is originated from the optimization added in the following PR https://github.com/moonbeam-foundation/moonbeam/pull/3212